src/parse.ts

```diff
@@ -65,7 +65,7 @@ export function parseHost (input: string = ''): ParsedHost {
 }
 
 export function stringifyParsedURL (parsed: ParsedURL) {
-  const fullpath = parsed.pathname + (parsed.search ? '?' + parsed.search : '') + parsed.hash
+  const fullpath = parsed.pathname + (parsed.search ? (parsed.search.includes('?') ? '' : '?') + parsed.search : '') + parsed.hash
```

Comment:
```suggestion
  const fullpath = parsed.pathname + (parsed.search ? (parsed.search.endsWith('?') ? '' : '?') + parsed.search : '') + parsed.hash
```

Reply:
```javascript
{
  pathname: '/',
  search: '?query=value',
  hash: '#test'
}
```

With `endsWith` this won't be ever true, maybe you meant `startsWith` right?

Thanks!

Reply:
Thanks yes you are right it should be with startsWith :)

Reply:
No problem, can we expect a new release soon?

I'd like to fix an issue with the i18n library and this function will make things easier and better.

Thanks!